test/unittests/test_dd_synthesis.cpp

```diff
@@ -0,0 +1,551 @@
+#include "core/truthTable/dd_synthesis.hpp"
+#include "core/truthTable/truth_table.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace dd::literals;
+using namespace syrec;
+
+class DDSynth: public testing::TestWithParam<std::string> {
```

Comment:
Again, why is this a parameterised test suite without parameterised tests?
Also: The class name should probably contain the word `test`.